src/tests/suite/daos_container.c

```diff
@@ -2203,6 +2204,52 @@ co_rf_simple(void **state)
 	test_teardown((void **)&arg);
 }
 
+static void
+delet_container_during_aggregation(void **state)
+{
+	test_arg_t	*arg = *state;
+	daos_obj_id_t	 oid;
+	daos_pool_info_t pinfo;
+	int		i;
+	int		rc;
+
+	/* Prepare records */
+	oid = dts_oid_gen(OC_RP_2G1, 0, arg->myrank);
+
+	print_message("Initial Pool Query\n");
+	pool_storage_info(state, &pinfo);
+
+	/* Aggregation will be Hold */
+	daos_debug_set_params(arg->group, -1, DMG_KEY_FAIL_LOC,
+			      DAOS_CONT_AGG_YIED | DAOS_FAIL_ALWAYS, 0, NULL);
+
+	/* Write/fetch and Punch Data with 2K size */
+	for (i = 0; i <= 50; i++)
+		io_simple_internal(state, oid, IO_SIZE_SCM * 32, DAOS_IOD_ARRAY,
+				   "io_simple_scm_array dkey",
+				   "io_simple_scm_array akey");
+
+	/**
+	 * Run Pool query every 5 seconds for Total 40 seconds
```

Comment:
Do we really need to wait that long? 25 seconds isn't enough?

Reply:
Yes, I wanted to run top of aggregation time to be sure it's about to start. And is it 25 or 30 seconds?

Reply:
I think 30 seconds should be enough.